persistence-cassandra/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/CassandraPersistenceSpec.scala

```diff
@@ -20,7 +20,8 @@ class CassandraPersistenceSpec private (system: ActorSystem) extends ActorSystem
     this(ActorSystem(testName, ActorSystemSetup(
       BootstrapSetup(TestUtil.persistenceConfig(
         testName,
-        CassandraLauncher.randomPort
+        CassandraLauncher.randomPort,
+        offsetStoreAutoCreate = false
```

Comment:
Hi @jroper! Thanks for comment. Yes, you are right. I've found it a 4:00 AM and left this issue for today's evening :D I left some message in "contributors" channel. I think I would need to re-factor current com.lightbend.lagom.scaladsl.persistence.cassandra.CassandraPersistenceSpec a bit.